django/contrib/admin/templates/admin/related_widget_wrapper.html

```diff
@@ -0,0 +1,21 @@
+{% load i18n admin_static %}
+<div class="related-widget-wrapper">
+    {{ widget }}
+    {% block links %}
+        {% if can_change_related %}
+        <a class="related-widget-wrapper-link change-related" id="change_id_{{ name }}" data-href-template="{{ change_related_template_url }}?{{ url_params }}" title="{% blocktrans %}Change selected {{ model }}{% endblocktrans %}">
```

Comment:
For new code, I'd like if you could limit the line lengths to 120 characters as it makes review in GitHub a lot easier not to have to scroll horizontally.